setup.py

```diff
@@ -32,6 +32,9 @@
 import os
 import pkgutil
 import sys
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), 'lib'))
+
 import vsc.install.shared_setup as shared_setup
 from vsc.install.shared_setup import vsc_setup, log, sdw
 import vsc.mympirun.mpi.mpi as mpim
```

Comment:
you have to replace the `subclasses(MPI)` call with a constant list of classes, and a add a unittest to make sure that this constant in setup.py is equal to the `subclasses(MPI)` result.
otherwise, you'll have to do some ugly hacks to make this work.